PWGJE/EMCALJetTasks/PWGJEEMCALJetTasksLinkDef.h

```diff
@@ -141,6 +141,7 @@
 #pragma link C++ namespace JETriggerRejectionAna+;
 #pragma link C++ class JETriggerRejectionAna::AliAnalysisTaskTriggerRejection+;
 #pragma link C++ class AliAnalysisTaskV0sInJetsEmcal+;
+#pragma link C++ class AliAnalysisTaskStrangenessInJets+;
```

Comment:
Since you make direct use of fastjet objects, you need to move this declaration under the `#ifdef HAVE_FASTJET` below